generated/nidigital/nidigital/session.py

```diff
@@ -2774,6 +2774,38 @@ def delete_all_time_sets(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def _file_helper(self, action, files):
+        if isinstance(files, str):
```

Comment:
"Ask for forgiveness, not for permission"

So you can try `[] + files`which raises `TypeError` if invoked with string; in which case you can use `append`